_action_files/fastpages.tpl

```diff
@@ -1,15 +1,32 @@
-{%- extends 'hide.tpl' -%}{% block body %}---
-{% if resources.toc != "" and resources.toc != nil %}toc: {{resources.toc}}{% endif %}
-{% if resources.title != "" and resources.title != nil %}title: {{resources.title}}{% endif %}
-{% if resources.image != "" and resources.image != nil %}image: {{resources.image}}{% endif %}
-keywords: {{resources.keywords}}
-sidebar: home_sidebar
-{% if resources.tags != "" and resources.tags != nil %}tags: {{resources.tags}}{% endif %}
-{% if resources.summary != "" and resources.summary != nil %}summary: "{{resources.summary}}"{% endif %}
+{%- extends 'hide.tpl' -%}
+{%- block body -%}
+{%- set internals = ["metadata", "output_extension", "inlining", 
+                    "raw_mimetypes", "global_content_filter"] -%}
 ---
-{% include 'autogen.tpl' %}
+{%- for k in resources |reject("in", internals) %}
+{{ k }}: {{ resources[k] }}
+{%- endfor %}
+---
+
+<!--
+#################################################
+### THIS FILE WAS AUTOGENERATED! DO NOT EDIT! ###
+#################################################
+# file to edit: {{ resources.nb_path }}
+-->
 
 <div class="container" id="notebook-container">
+        {% raw -%}
+           {% if page.badges == true or page.badges == "True"  or page.badges == "true" or page.badges == nil %}
```

Comment:
Since this is executed as Python, I think you can drop `page.badges == true`, since `true` in this case will be treated as an undefined variable (?). Haven't tested it yet though.

Reply:
So the stuff inside the `{% raw %} .... {% endraw %}` tags ends up being executed as `liquid`, but you are still right that this is redundant and confusing.  

I thought that this was needed [when experimenting with the liquid  playground](https://jumpseller.com/support/liquid-sandbox/) but it seems that Jekyll helpfully translates `True` and `true` to `true`.  So I can simplify this as you suggest.  Thank you for pointing it out